### ISSUE TITLE
✨ [User-requested Feature] Susceptibility Distortion Correction via SynBold-Disc

### ISSUE THREAD

**Issue by nimzodisaster** [FCP-INDI/C-PAC#2068]
### Related problem

Susceptibility Distortion Correction is a very useful technique, but topup scans and/or phasemap images haven't always been acquired successfully. I came across this method (SynBOLD-Disc) and have tried it one some data, and it appears to quite effective in our own data, **AND it does NOT require phasemap or topup images, just the EPI and T1.** 

The authors have published the source code.
https://github.com/MASILab/SynBOLD-DisCo/tree/main
https://www.sciencedirect.com/science/article/pii/S0730725X23001121

### Proposed feature

Add SynBOLD-Disc as an options to CPAC distortion correction.

### Acceptance criteria

SynBOLD-Disc based distortion correction in CPAC.

**Comment by sgiavasis** [comment]
Interesting, thank you for bringing this to our attention @nimzodisaster !

**Comment by nimzodisaster** [comment]
Absolutely. I am in the process of doing a comparison across all my data
that does have the distortion correction scans and/or topups.

I am also trying to figure out how I might include this into the CPAC
pipeline (which is why I cloned C-PAC recently).

My original thought was to simply run the SynBold-Disc pipeline, then use
its outputs as inputs into C-PAC. However, motion correction is performed
as part of the TOP-UP procedure, and does not produce motion parameters. I
was then unsure how I might get that to work in C-PAC. Looking at the
standard CPAC pipeline, it does seem like motion parameters and topup are
produced in parallel pipes, so my thought was to create a new nipype node
option, but although I do program; I do not have much experience in working
within larger projects..just writing one-offs.

-Best,

Joshua
